BrainPortal/app/controllers/data_providers_controller.rb

```diff
@@ -842,7 +993,8 @@ def data_provider_params #:nodoc:
       # they don't control.
       params.require_as_params(:data_provider).permit(
         :name, :description, :group_id, :time_zone,
-        :alternate_host,
+        :alternate_host, :remote_user, :remote_host, :alternate_host, :remote_dir,
```

Comment:
This introduces serious security issues that could allow users total control of all CBRAIN. Don't modify security code unless you understand the implications!